src/SummaryCell.tsx

```diff
@@ -11,12 +11,14 @@ interface SharedCellRendererProps<R, SR>
 
 interface SummaryCellProps<R, SR> extends SharedCellRendererProps<R, SR> {
   row: SR;
+  bottom: number;
```

Comment:
![image](https://user-images.githubusercontent.com/567105/149341471-0bf990db-1ac8-4b5a-9710-87b8b7c160f2.png)
Summary rows don't stick to the very bottom of the container anymore
It looks like adding `top` might be necessary.

Reply:
![image](https://user-images.githubusercontent.com/1452717/149391459-afcf7a48-efaf-47f7-a185-3989b30daa0f.png)

![image](https://user-images.githubusercontent.com/1452717/149391552-526573f2-bfbd-41f8-9467-26374f88b231.png)